### ISSUE TITLE
Presentation slide size is hard-coded and not honoring template size

### ISSUE THREAD

**Issue by jaakkor2** [ASML-Labs/PPTX.jl#34]
Hard-coding is here https://github.com/ASML-Labs/PPTX.jl/blob/v0.6.0/src/Presentation.jl#L156-L159
One could get the size of the template before it is over-written by something like this
```
    doc = readxml("ppt/presentation.xml")
    r = root(doc)
    n = findfirst("//p:sldSz", r)
    cx, cy = n["cx"], n["cy"]    
```

**Comment by matthijscox-asml** [comment]
Good catch, let me have a look at this.

**Comment by matthijscox-asml** [comment]
Should be fixed in v0.6.1
